stdlib/ballerina-http/src/main/ballerina/swagger/annotation.bal

```diff
@@ -15,14 +15,18 @@
 // under the License.
 
 
-@Description {value: "Model for additional swagger information of a ballerina service"}
-@Field {value: "title: Title of the swagger definition"}
-@Field {value: "serviceVersion: Version of the swagger API"}
-@Field {value: "termsOfService: Service usage terms and conditions"}
-@Field {value: "contact: Contact information for the exposed API."}
-@Field {value: "license: License information for the exposed API."}
-@Field {value: "externalDoc: Additional external documentation."}
-@Field {value: "tags: A list of tags used by the specification with additional metadata"}
+documentation {
+    Model for additional swagger information of a ballerina service.
```

Comment:
Change "swagger" to "Swagger" and "ballerina" to "Ballerina" in descriptions throughout the documentation sections in this file